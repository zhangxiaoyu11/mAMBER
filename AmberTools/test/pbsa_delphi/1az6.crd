    
  489
  -8.1336485  -8.9056748  -3.6763895  -8.5405637  -7.9928081  -3.5308065
  -7.7923848  -9.2809925  -2.8030190  -8.8297987  -9.5367844  -4.0467618
  -7.0398042  -8.8438389  -4.6615475  -6.1396691  -8.4764027  -4.1687519
  -6.7243597 -10.2525265  -5.1941864  -6.1416298 -10.1852053  -6.1128777
  -5.9564337 -11.0797844  -4.1527061  -6.6165349 -11.3961101  -3.3450526
  -5.5392860 -11.9640837  -4.6344764  -5.1365045 -10.4963270  -3.7339062
  -7.9437270 -10.9144217  -5.4182701  -7.7772599 -11.6198670  -6.0477427
  -7.4054443  -7.8629394  -5.7990859  -8.0403157  -8.2338277  -6.7771303
  -7.0347254  -6.5576089  -5.6247236  -6.4841313  -6.3725162  -4.7984749
  -7.3652195  -5.3804218  -6.4495176  -7.8286394  -5.7259435  -7.3736157
  -8.3864117  -4.5060573  -5.6827218  -9.2733041  -5.1043394  -5.4739627
  -7.9521975  -4.2032513  -4.7299008  -8.8123965  -3.2362050  -6.4628006
  -9.5463487  -2.6969058  -5.8639929  -7.9527172  -2.5817414  -6.6067760
  -9.4500261  -3.5742579  -7.8257339 -10.5010433  -4.1891758  -7.8999485
  -8.8192316  -3.2164036  -8.9576657  -9.3354036  -3.4885730  -9.7820384
  -7.9434231  -2.7146314  -8.9935018  -6.0964063  -4.5613034  -6.8106337
  -5.4193306  -4.0183627  -5.9405335  -5.7923271  -4.4447958  -8.1288957
  -6.4613749  -4.8296563  -8.7803229  -4.6347990  -3.8321633  -8.8096605
  -3.7322380  -4.3632560  -8.5072961  -4.8179075  -4.0528604 -10.3285586
  -4.1335148  -3.4137466 -10.8864477  -4.5952296  -5.0933793 -10.5648546
  -6.1603922  -3.7850272 -10.6974226  -6.2076663  -3.7175408 -11.6538800
  -4.4036980  -2.3300709  -8.4696870  -4.6863160  -1.4343359  -9.2619604
  -3.8021156  -2.0469302  -7.2721283  -3.6091546  -2.8429417  -6.6811698
  -3.2943973  -0.7191679  -6.8756445  -3.9814810   0.0408268  -7.2476986
  -3.2001747  -0.5497431  -5.3433190  -2.5621629  -1.3183157  -4.9070427
  -2.7256598   0.4098364  -5.1380302  -4.5305468  -0.5591099  -4.6495415
  -5.4788759  -1.5574897  -4.7092390  -5.3937202  -2.4236209  -5.2217478
  -6.4543831  -1.2712421  -3.8411976  -7.2530678  -1.9361295  -3.5472120
  -6.1502695  -0.1139620  -3.2437236  -6.5817132   0.2090736  -2.3895548
  -4.9618813   0.3596253  -3.7402594  -4.4340545   1.2366685  -3.3958844
  -1.9222625  -0.4736299  -7.5260144  -0.9089242  -1.0212589  -7.0911150
  -1.9032767   0.3873100  -8.5805980  -2.7986638   0.7197132  -8.9090648
  -0.6986544   0.8720547  -9.2637147  -0.0998769   0.0101784  -9.5582231
  -1.1292058   1.6050390 -10.5491803  -1.7485096   2.4664827 -10.2992235
  -0.2460285   1.9433641 -11.0910605  -1.6971684   0.9266057 -11.1857702
   0.1791492   1.7360898  -8.3041461   0.1083074   2.9623324  -8.2856379
   1.0061223   1.0494287  -7.4606199   0.8600081   0.0500571  -7.4632832
   1.9222734   1.6023459  -6.4567184   2.7987758   0.9593550  -6.3766792
   2.2483255   2.5963133  -6.7630178   1.2576636   1.6937184  -5.0715503
   1.4668218   0.8282915  -4.2222662   0.4709848   2.7876404  -4.8328113
   0.3013720   3.3978531  -5.6195590  -0.1125278   3.1513172  -3.5295331
   0.6451333   2.9758162  -2.7658253  -0.4476484   4.6673922  -3.5017784
   0.4105466   5.2222104  -3.8809592  -1.2993305   4.8713470  -4.1507319
  -0.7589133   5.1644203  -2.0654859  -1.6861283   4.7034715  -1.7250471
   0.0446379   4.8498031  -1.3995865  -0.9026157   6.7009845  -1.9526371
  -0.6669313   7.4568873  -2.8836461  -1.2875673   7.2082608  -0.7713463
  -1.3644635   8.2141870  -0.7233930  -1.4641182   6.5664581  -0.0117286
  -1.3112023   2.2355828  -3.1689519  -2.4732143   2.5763485  -3.3701571
  -0.9993072   1.0389522  -2.5941089  -0.0151786   0.8141717  -2.5614810
  -1.9109049   0.1557074  -1.8543774  -2.7279719  -0.1525525  -2.5066637
  -1.1510541  -1.1190231  -1.4233965  -1.7795354  -1.7078416  -0.7552616
  -0.9096978  -1.7149144  -2.3036012   0.3922536  -0.6867350  -0.5845363
   0.6016568  -1.8613661   0.0165369  -2.5734231   0.9167140  -0.6705866
  -3.7966952   0.9178269  -0.5709304  -1.7537152   1.5945052   0.2050411
  -0.7654740   1.5275407   0.0075691  -2.1898138   2.3420301   1.3973533
  -3.2356560   2.6268112   1.2824019  -2.1157100   1.6851580   2.2640295
  -1.3796916   3.6362661   1.6748225  -1.5527085   4.6528340   0.9996345
  -0.5302156   3.5977350   2.7565188  -0.4576552   2.7080904   3.2291374
  -0.0168641   4.7909094   3.4608833   0.9757267   4.5935931   3.8657658
   0.0377815   5.6382947   2.7774707  -0.9704654   5.1648965   4.6136911
  -2.1151551   5.5370095   4.3715481  -0.5247226   4.9596576   5.8938953
   0.4424239   4.6930959   6.0108256  -1.3639829   5.0137062   7.1142598
  -2.1179098   4.2310922   7.0293505  -0.5076253   4.7002837   8.3810414
   0.3098400   5.4188970   8.4397432  -1.3685931   4.8286415   9.6667203
  -2.1922032   4.1151149   9.6413012  -0.7581228   4.6402506  10.5498601
  -1.7665141   5.8391442   9.7596878   0.0903088   3.2588945   8.2572422
  -0.7230130   2.5380118   8.1739615   0.6879371   3.1916639   7.3481655
   1.0021117   2.8378544   9.4285275   0.4165642   2.6992986  10.3373930
   1.4856763   1.8928603   9.1810372   1.7682394   3.5952906   9.5941763
  -2.1309492   6.3634678   7.1915248  -1.5227499   7.4287390   7.2495902
  -3.5029703   6.2888430   7.1472590  -3.8963450   5.3588791   7.1701424
  -4.4325967   7.4208188   6.9258404  -5.2272714   7.3850901   7.6710354
  -3.9011233   8.3653048   7.0423857  -5.0970717   7.4166606   5.5188109
  -5.7985218   8.3589168   5.1661778  -4.8729424   6.3289553   4.7192044
  -4.1614339   5.7039069   5.0701610  -5.3626786   5.9768810   3.3744410
  -4.7684167   6.5425159   2.6567991  -5.0802108   4.4633293   3.1116877
  -5.7385992   4.1106642   2.3178035  -4.0614554   4.3879475   2.7314843
  -5.2061577   3.5272562   4.3131599  -6.4738663   3.1531678   4.8017791
  -7.3674358   3.4878116   4.2958668  -6.5922520   2.3406834   5.9446832
  -7.5724975   2.0731111   6.3106392  -5.4312315   1.8699663   6.5938406
  -5.5359183   1.0628931   7.6830767  -6.4471409   0.9193276   7.9489014
  -4.1577574   2.2091580   6.0882999  -3.2750793   1.8244987   6.5774999
  -4.0484127   3.0352695   4.9527275  -3.0714334   3.2844549   4.5656643
  -6.8490022   6.3586844   3.0965253  -7.7804986   5.7479915   3.6160990
  -7.0841655   7.3411390   2.1686344  -6.2880968   7.8824027   1.8630059
  -8.4199137   7.7807357   1.6938927  -9.0881788   7.8479051   2.5523856
  -8.3177214   9.1895891   1.0670859  -7.7272643   9.1369395   0.1523791
  -9.3180884   9.5464049   0.8220365  -7.7062725  10.0938691   1.9647139
  -7.7169001  10.9732933   1.5798759  -9.0655688   6.7771543   0.6879732
  -9.5312604   7.1658221  -0.3814136  -9.1003754   5.4578727   1.0476950
  -8.6986648   5.2508681   1.9509544  -9.6163839   4.3172464   0.2736977
 -10.7041374   4.3197552   0.3435969  -9.3269130   4.4329430  -0.7707739
  -9.0872721   2.9436955   0.7820135  -8.3119229   2.8914004   1.7371582
  -9.5312590   1.7982658   0.1382052 -10.5280261   1.7390901  -0.9335774
 -10.0195087   1.7762692  -1.8969713 -11.2621618   2.5415946  -0.8619457
 -11.2213383   0.3844738  -0.7632173 -11.6433704   0.0249026  -1.7016722
 -11.9944118   0.4650634   0.0009578 -10.0896592  -0.5208629  -0.2605320
  -9.5503180  -0.9256222  -1.1169086 -10.4756838  -1.3322631   0.3564894
  -9.1814633   0.4279187   0.5567060  -9.4342272   0.3243282   1.6119215
  -7.6787672   0.0911525   0.3568900  -7.1646021   0.1385639  -0.7605212
  -6.9975277  -0.3080743   1.4720947  -7.4900685  -0.2064588   2.3479816
  -5.5731994  -0.6490006   1.6974983  -4.9648955   0.2050464   1.3997233
  -5.3433485  -0.9181648   3.2148020  -5.4812268  -1.9803096   3.4171339
  -3.9385636  -0.5062224   3.6756196  -3.7901779   0.5594617   3.5012666
  -3.8208236  -0.7216676   4.7376087  -3.1901278  -1.0718687   3.1206528
  -6.2791928  -0.2134966   4.0019420  -5.8652806  -0.0159514   4.8453003
  -5.0846186  -1.8955771   0.9170900  -3.8909789  -2.0472057   0.6610229
  -6.0148998  -2.8596298   0.6187812  -6.9701879  -2.6218227   0.8445527
  -5.7220196  -4.2033231   0.0703700  -4.8084348  -4.5526814   0.5514226
  -6.8418855  -5.2196955   0.4483658  -6.9146859  -5.2288575   1.5358934
  -8.2455708  -4.8707822  -0.1035488  -8.2611990  -4.9522737  -1.1903856
  -8.9777060  -5.5656160   0.3078927  -8.5285649  -3.8586369   0.1855452
  -6.4643776  -6.6491088   0.0012797  -5.4883865  -6.9173480   0.4057409
  -7.2021290  -7.3556884   0.3815009  -6.4354070  -6.7157541  -1.0862950
  -5.4323614  -4.1553546  -1.4478522  -6.2822576  -3.7448493  -2.2379662
  -4.2014623  -4.6002154  -1.8669702  -3.5452616  -4.9054763  -1.1624724
  -3.8518639  -4.7228301  -3.2926349  -4.4707184  -4.0038016  -3.8294004
  -2.3886407  -4.2893263  -3.5079110  -2.2005717  -3.3738306  -2.9470209
  -1.7026326  -5.0642635  -3.1659092  -2.1485437  -3.9583800  -5.2763867
  -0.8622493  -3.6070222  -5.1933686  -4.2306489  -6.1294392  -3.8704066
  -5.2477715  -6.7020828  -3.4813538  -3.4421589  -6.6677137  -4.8575873
  -2.6620725  -6.0781738  -5.1105853  -3.6731917  -7.8565961  -5.6968405
  -4.6437615  -8.2896250  -5.4548067  -3.7023566  -7.4232887  -7.1780901
  -2.7665111  -6.9339327  -7.4479492  -3.8393742  -8.2986937  -7.8129068
  -4.5307745  -6.7357649  -7.3487792  -2.6377332  -8.9843920  -5.4454983
  -3.0303149 -10.0894259  -5.0862998  -1.3121536  -8.7109893  -5.6631739
  -1.0489942  -7.7688368  -5.9145621  -0.1745915  -9.6327403  -5.4776164
  -0.2497445 -10.0707143  -4.4823120  -0.2410192 -10.7853914  -6.5087803
  -1.2414295 -11.2181347  -6.5123295  -0.0138694 -10.4051688  -7.5047394
   0.6810830 -11.7962667  -6.1549663   0.4969303 -12.5840064  -6.6718357
   1.1708290  -8.8539744  -5.5225792   1.1720513  -7.6268107  -5.4877502
   2.3066916  -9.6058024  -5.5634883   2.0986585 -10.5913732  -5.6374644
   3.7065928  -9.2962203  -5.2586507   4.3267624  -9.4598941  -6.1399569
   3.7870921  -8.2555595  -4.9445738   4.1553818 -10.2648995  -4.1357496
   4.0481467 -11.4801848  -4.2928971   4.5791983  -9.7177296  -2.9621503
   4.7958554  -8.7343657  -3.0406048   4.7574185 -10.3866213  -1.6549759
   4.3423696 -11.3903951  -1.7459297   6.2629725 -10.5849148  -1.3184847
   6.6855381 -11.2381311  -2.0819288   7.1316650  -9.3198237  -1.2673856
   6.7655289  -8.6362433  -0.5013817   8.1653256  -9.5869325  -1.0476161
   7.1089308  -8.8212834  -2.2364263   6.4079612 -11.2260011  -0.0745080
   7.3479374 -11.3301569   0.0904068   3.9036969  -9.7375803  -0.5230015
   3.1660490 -10.4473442   0.1586544   3.9598252  -8.3800242  -0.3136650
   4.5787385  -7.8311563  -0.8931392   3.3464288  -7.6207652   0.8001522
   2.5145037  -8.2043638   1.1943775   4.3781055  -7.3994241   1.9485809
   3.8629350  -6.9404839   2.7924253   5.0545745  -8.6809239   2.4567035
   5.6954230  -9.1026220   1.6823754   5.6593770  -8.4521675   3.3341910
   4.2922917  -9.4116428   2.7270259   5.3848497  -6.5179301   1.5250570
   5.9758694  -6.3299664   2.2578362   2.7704735  -6.2284956   0.3921332
   2.9001065  -5.7355642  -0.7279110   2.1174904  -5.5578526   1.3931434
   2.0465144  -6.0489193   2.2728683   1.8403143  -4.1156153   1.4482909
   1.8532472  -3.6917619   0.4441587   0.4331508  -3.9413743   2.0650716
  -0.3038769  -4.3724647   1.3875384   0.3882323  -4.4715043   3.0164103
   0.0324611  -2.2000883   2.3556230  -1.2541978  -2.3858551   2.6636756
   2.9838549  -3.4653378   2.2731984   2.9820677  -3.5532179   3.5003011
   4.0183209  -2.8942639   1.5796964   3.9181023  -2.7667377   0.5828046
   5.3541279  -2.6179223   2.1233087   5.4138121  -3.1383890   3.0791605
   6.4299542  -3.1960654   1.1679431   6.1009844  -4.1622323   0.7853199
   6.5498054  -2.5232217   0.3188157   7.8007216  -3.3818220   1.8616943
   8.5044560  -3.8101729   1.1479904   8.1847599  -2.4094319   2.1700328
   7.7078365  -4.2961464   3.0996388   7.5063984  -3.8368841   4.2146466
   7.8174161  -5.6293196   2.9381617   7.8781887  -6.1229956   3.8171905
   8.0873356  -6.0062875   2.0408666   5.6084058  -1.1169618   2.3954061
   5.4175780  -0.2574410   1.5331985   6.0998389  -0.8250037   3.6358009
   6.2893001  -1.6230413   4.2251568   6.4698127   0.5031111   4.1550943
   5.6662627   1.1907025   3.8912279   6.5598960   0.4685853   5.7110967
   5.6014200   0.1076699   6.0841436   7.6552476  -0.4719015   6.2690477
   8.6455119  -0.1153853   5.9855470   7.5922844  -0.5028991   7.3567861
   7.5193756  -1.4833067   5.8860358   6.7674356   1.8831990   6.2872003
   5.9722365   2.5366916   5.9284442   6.7283483   1.8428438   7.3757515
   7.7359140   2.2813665   5.9845241   7.7650990   1.0353077   3.4932620
   8.7845263   0.3508454   3.4404232   7.7130924   2.3113468   3.0061783
   6.8001719   2.7431163   3.0220300   8.8572657   3.1460929   2.5941950
   9.7310717   2.5213232   2.4091915   8.5075077   3.9435140   1.3049203
   7.5080254   4.3672958   1.4026145   9.2124776   4.7713436   1.2286569
   8.5930943   3.1449914  -0.0252688   9.6112261   2.7766860  -0.1512007
   7.6245296   1.9510836  -0.0983147   6.6152344   2.2682542   0.1640318
   7.6313173   1.5345437  -1.1055627   7.9452084   1.1685523   0.5893691
   8.2732043   4.1009747  -1.1910247   8.9742982   4.9355758  -1.1880653
   8.3693208   3.5690612  -2.1375603   7.2564177   4.4821892  -1.0965717
   9.2118346   4.0937744   3.7672463  10.2793071   4.0079471   4.3639276
   8.2415089   4.9866069   4.1122666   7.3852125   4.8867546   3.5860670
   8.0900602   5.7818103   5.3372859   8.8846914   5.5341449   6.0410783
   8.1503474   7.2869299   4.9741023   7.4502702   7.5048784   4.1675724
   7.8564407   7.8753381   5.8432942   9.5574666   7.7368684   4.5607551
  10.3005521   8.2844906   5.3592543   9.9582143   7.5309488   3.2943886
  10.9076404   7.8321096   3.1270793   9.3786777   7.0357814   2.6317827
   6.7128313   5.3916013   5.9446226   5.8121024   5.0771557   5.1718804
   6.5006324   5.4104059   7.3116258   7.4199927   5.9108251   8.3322970
   7.9885068   6.7727545   7.9830559   8.0956332   5.1102012   8.6333156
   6.5256104   6.3079126   9.5114305   6.1380495   7.3137283   9.3494690
   7.0613220   6.2542322  10.4591814   5.3723581   5.2915146   9.4488714
   4.4466724   5.7213272   9.8315816   5.6393511   4.4069742  10.0271480
   5.2631026   4.9078360   7.9568276   5.2530373   3.8200604   7.8879581
   3.9399010   5.3914781   7.2944043   2.9683442   4.6425014   7.2143949
   3.9145298   6.6593693   6.7831374   4.7573413   7.2030370   6.9022771
   2.7476401   7.2942566   6.1430322   1.8470481   6.8397130   6.5558570
   2.7164782   8.7818268   6.5639721   3.6466618   9.2642675   6.2637900
   1.8980053   9.2798638   6.0442012   2.5143757   8.9457541   8.0657390
   3.6192470   8.9966860   8.9421295   4.6234853   8.9898788   8.5448145
   3.4211192   9.0351744  10.3372680   4.2705559   9.0549915  11.0039500
   2.1089246   9.0412636  10.8631007   1.9089822   9.0660829  12.2104899
   2.7275317   9.0741805  12.7119989   1.0030557   9.0165855   9.9855777
   0.0007092   9.0243549  10.3876240   1.2091763   8.9652885   8.5939727
   0.3592958   8.9249425   7.9287840   2.6304758   7.0330345   4.6013492
   1.6312100   7.4137548   3.9991707   3.6094851   6.3019500   3.9627482
   4.3687764   5.9786019   4.5450048   3.5833839   5.7330098   2.5973917
   2.5821551   5.8804878   2.1925569   4.5705855   6.4599501   1.6409197
   4.5100462   7.5297827   1.8406525   5.5929635   6.1483049   1.8547444
   4.2632630   6.2185640   0.1630297   4.2590834   4.9136047  -0.3819315
   4.5908828   4.0677378   0.2018660   3.7882580   4.6862519  -1.6885467
   3.7035506   3.6786559  -2.0679871   3.4223379   5.7751856  -2.5034722
   2.9866346   5.5407646  -3.7718523   2.7176006   4.6294244  -3.9085262
   3.5090049   7.0904084  -1.9965184   3.2260260   7.9234748  -2.6228757
   3.9030303   7.3050099  -0.6605372   3.9122185   8.3092178  -0.2631927
   3.8736566   4.2042615   2.6133793   5.0136608   3.7638332   2.7896140
   2.8213211   3.3872795   2.2996591   1.9407624   3.8300131   2.0789709
   2.9555829   1.9647480   1.9467033   3.9968524   1.6731751   2.0839734
   2.1170242   1.0827649   2.8699071   2.3198866   1.3373596   3.9101611
   1.0563926   1.2087970   2.6524750   2.5113229  -0.2442560   2.6293698
   1.9029387  -0.8511794   3.0572864   2.5994595   1.7241050   0.4606799
   1.6422101   2.2984508  -0.0645817   3.4361519   0.8843201  -0.2148092
   4.1175484   0.4214378   0.3696040   3.5122579   0.5648157  -1.6439103
   2.6602214   1.0420874  -2.1280036   4.8275491   1.1766549  -2.1996883
   5.0761691   2.0677320  -1.6232600   5.6442807   0.4678837  -2.0629665
   4.7377189   1.5919498  -3.6873178   4.6837549   0.6970709  -4.3073015
   3.8373769   2.1844312  -3.8499509   5.9698354   2.4230013  -4.1083071
   7.0731955   1.9126697  -4.2176266   5.8361213   3.7440271  -4.3530464
   6.7146568   4.1594072  -4.6282456   4.9752167   4.2652516  -4.2677926
   3.4315621  -0.9766227  -1.8850975   3.8851043  -1.7856274  -1.0749353
   2.8119417  -1.3796112  -3.0299499   2.4328103  -0.6323237  -3.5937945
   2.5887391  -2.7538075  -3.5002890   2.2507041  -3.3691782  -2.6665314
   1.4761597  -2.7139832  -4.5648308   0.5786712  -2.2456616  -4.1607400
   1.8232630  -2.1430038  -5.4259938   1.1224772  -4.4154718  -5.0662415
   0.9080661  -4.1657783  -6.3610689   3.8946106  -3.3913172  -4.0560067
   4.2200758  -3.2117456  -5.2301026   4.6353353  -4.1369278  -3.1736436
   4.2409075  -4.2508773  -2.2508531   5.8976122  -4.8726424  -3.3912060
   6.1266964  -4.9137850  -4.4560664   7.0526561  -4.1396034  -2.6255447
   6.6490802  -3.7396688  -1.6953411   7.7951185  -4.8901350  -2.3543397
   7.8098111  -3.0064940  -3.3811041   8.3426545  -3.4420151  -4.2263847
   6.9134803  -1.8718095  -3.9073719   6.1912938  -1.5843950  -3.1432125
   7.5159301  -1.0056933  -4.1812273   6.3956515  -2.2132693  -4.8036745
   8.8495737  -2.3846294  -2.4280532   9.5114770  -3.1605310  -2.0433963
   9.4471326  -1.6495790  -2.9672417   8.3434066  -1.8926190  -1.5974982
   5.7139052  -6.3584996  -2.9503212   5.6139913  -7.2279246  -3.8415188
   5.6625480  -6.6263025  -1.7328674
