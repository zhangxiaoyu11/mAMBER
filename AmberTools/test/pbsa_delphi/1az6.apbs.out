

----------------------------------------------------------------------
	APBS -- Adaptive Poisson-Boltzmann Solver
	Version 1.2.1
	
	Nathan A. Baker (baker@biochem.wustl.edu)
	Dept. Biochemistry and Molecular Biophysics
	Center for Computational Biology
	Washington University in St. Louis
	
	Additional contributing authors listed in the code documentation.
	
	Copyright (c) 2002-2009, Washington University in St. Louis.
	Portions Copyright (c) 2002-2009.  Nathan A. Baker
	Portions Copyright (c) 1999-2002.  The Regents of the University of California.
	Portions Copyright (c) 1995.  Michael Holst
	
	All rights reserved.
	
	Redistribution and use in source and binary forms, with or without
	modification, are permitted provided that the following conditions are met: 
	
	* Redistributions of source code must retain the above copyright notice, this
	list of conditions and the following disclaimer.  
	
	* Redistributions in binary form must reproduce the above copyright notice,
	this list of conditions and the following disclaimer in the documentation
	and/or other materials provided with the distribution.
	
	* Neither the name of Washington University in St. Louis nor the names of its
	contributors may be used to endorse or promote products derived from this
	software without specific prior written permission.
	
	THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
	"AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
	LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
	A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
	CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
	EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
	PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
	PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
	LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
	NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
	SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
----------------------------------------------------------------------
	APBS uses FETK (the Finite Element ToolKit) to solve the
	Poisson-Boltzmann equation numerically.  FETK is a portable collection
	of finite element modeling class libraries developed by the Michael Holst
	research group and written in an object-oriented form of C.  FEtk is
	designed to solve general coupled systems of nonlinear partial differential
	equations using adaptive finite element methods, inexact Newton methods,
	and algebraic multilevel methods.  More information about FEtk may be found
	at <http://www.FEtk.ORG>.
----------------------------------------------------------------------
	APBS also uses Aqua to solve the Poisson-Boltzmann equation numerically.  
	Aqua is a modified form of the Holst group PMG library <http://www.FEtk.ORG>
	which has been modified by Patrice Koehl
	<http://koehllab.genomecenter.ucdavis.edu/> for improved efficiency and
	memory usage when solving the Poisson-Boltzmann equation.
----------------------------------------------------------------------
	Please cite your use of APBS as:

	Baker NA, Sept D, Joseph S, Holst MJ, McCammon JA. Electrostatics of
	nanosystems: application to microtubules and the ribosome. Proc.
	Natl. Acad. Sci. USA 98, 10037-10041 2001.
	

This executable compiled on Nov 30 2009 at 16:33:06

Parsing input file apbs.in...
Parsed input file.
Got paths for 1 molecules
Reading PQR-format atom data from 1az6.pqr0.
  489 atoms
  Centered at (-5.430e-001, -8.055e-001, 5.290e-001)
  Net charge 1.00e+000 e
Preparing to run 2 PBE calculations.
----------------------------------------
CALCULATION #1 (1az6-solv): MULTIGRID
  Setting up problem...
  Vpbe_ctor:  Using max ion radius (2 A) for exclusion function
  Debye length:  7.95385 A
  Current memory usage:  453.280 MB total, 453.280 MB high water
  Using linear spline charge discretization.
  Grid dimensions: 129 x 129 x 129
  Grid spacings: 0.500 x 0.500 x 0.500
  Grid lengths: 64.000 x 64.000 x 64.000
  Grid center: (-0.543, -0.805, 0.529)
  Multigrid levels: 6
  Molecule ID: 1
  Linearized traditional PBE
  Single Debye-Huckel sphere boundary conditions
  2 ion species (0.150 M ionic strength):
    2.000 A-radius, 1.000 e-charge, 0.150 M concentration
    2.000 A-radius, -1.000 e-charge, 0.150 M concentration
  Solute dielectric: 1.000
  Solvent dielectric: 80.000
  Using "molecular" surface definition; no smoothing
  Solvent probe radius: 1.500 A
  Temperature:  300.000 K
  Electrostatic energies will be calculated
  Solving PDE (see io.mc* for details)...
  Calculating energy (see io.mc* for details)...
  Total electrostatic energy = 8.717303205204E+004 kJ/mol
  Calculating forces...
----------------------------------------
CALCULATION #2 (1az6-ref): MULTIGRID
  Setting up problem...
  Vpbe_ctor:  Using max ion radius (2 A) for exclusion function
  Debye length:  0.889267 A
  Current memory usage:  453.280 MB total, 458.444 MB high water
  Using linear spline charge discretization.
  Grid dimensions: 129 x 129 x 129
  Grid spacings: 0.500 x 0.500 x 0.500
  Grid lengths: 64.000 x 64.000 x 64.000
  Grid center: (-0.543, -0.805, 0.529)
  Multigrid levels: 6
  Molecule ID: 1
  Linearized traditional PBE
  Single Debye-Huckel sphere boundary conditions
  2 ion species (0.150 M ionic strength):
    2.000 A-radius, 1.000 e-charge, 0.150 M concentration
    2.000 A-radius, -1.000 e-charge, 0.150 M concentration
  Solute dielectric: 1.000
  Solvent dielectric: 1.000
  Using "molecular" surface definition; no smoothing
  Solvent probe radius: 1.500 A
  Temperature:  300.000 K
  Electrostatic energies will be calculated
  Solving PDE (see io.mc* for details)...
  Calculating energy (see io.mc* for details)...
  Total electrostatic energy = 8.851426391151E+004 kJ/mol
  Calculating forces...
----------------------------------------
PRINT STATEMENTS

print energy 1 (1az6-solv) - 2 (1az6-ref) end
  Local net energy (PE 0) = -1.341231859473E+003 kJ/mol
  Global net ELEC energy = -1.341231859473E+003 kJ/mol
----------------------------------------
CLEANING UP AND SHUTTING DOWN...
Destroying force arrays.
No energy arrays to destroy.
Destroying multigrid structures.
Destroying 1 molecules
Final memory usage:  0.001 MB total, 458.444 MB high water


Thanks for using APBS!

